teams/archive/community-ctcft.toml

```diff
@@ -2,17 +2,14 @@ name = "community-ctcft"
 subteam-of = "community"
 
 [people]
-leads = [
-    "AngelOnFira",
-    "nikomatsakis",
-]
-members = [
+leads = []
+members = []
+alumni = [
     "AngelOnFira",
     "nikomatsakis",
     "technetos",
-    "yaahc",
+    "yaahc"
```

Comment:
no training comma? :c

Reply:
Probably I didn't *train* my TOML formatter well enough, need more training data.

Reply:
in my defense I haven't woken up yet when writing this